sam-groups/actions/register/app.js

```diff
@@ -86,9 +86,64 @@ exports.handler = async (event) => {
     })
   })
 
-  await snsClient.send(publishCommand)
+  const promises = [snsClient.send(publishCommand)]
+
+  if (user !== undefined && user.group !== undefined) {
+    promises.push(informGroup(id, user.group))
+  }
+
+  await Promise.allSettled(promises)
 
   return {
     statusCode: 200
   }
 }
+
+// ===== ==== ====
+// HELPERS
+
+async function informGroup (userId, groupId) {
+  // retreive group
+  const getGroupCommand = new GetCommand({
+    TableName: GROUPS_TABLE_NAME,
+    Key: { id: groupId },
+    ProjectionExpression: '#id, #users',
+    ExpressionAttributeNames: {
+      '#id': 'id',
+      '#users': 'users'
+    }
+  })
+  const group = await dynamoDBDocumentClient.send(getGroupCommand).then((response) => (response.Item))
```

Comment:
I don't understand why here you do not have to put the condition like this:

if (response.Count > 0) {
      return response.Items[0]
    } else {
      return undefined
    }

while below you are testing the group == undefined case ?

Reply:
Je note, je t'expliquerai de vive-voix 😊